app/src/main/java/de/westnordost/streetcomplete/quests/toilets_fee/AddToiletsFee.java

```diff
@@ -0,0 +1,51 @@
+package de.westnordost.streetcomplete.quests.toilets_fee;
+
+import android.os.Bundle;
+
+import javax.inject.Inject;
+
+import de.westnordost.streetcomplete.R;
+import de.westnordost.streetcomplete.data.QuestImportance;
+import de.westnordost.streetcomplete.data.osm.SimpleOverpassQuestType;
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder;
+import de.westnordost.streetcomplete.data.osm.download.OverpassMapDataDao;
+import de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment;
+import de.westnordost.streetcomplete.quests.YesNoQuestAnswerFragment;
+
+public class AddToiletsFee extends SimpleOverpassQuestType
+{
+	@Inject public AddToiletsFee(OverpassMapDataDao overpassServer)
+	{
+		super(overpassServer);
+	}
+
+	@Override
+	protected String getTagFilters()
+	{
+		return "nodes, ways, relations with amenity=toilets and !fee";
```

Comment:
I agree and will remove "relations" from query :+1: 
And yes, I paint the icon by myself. I'm not a good drawer, so feel free to draw another icon.